scalafmt-tests/src/test/resources/newlines/source_unfold.stat

```diff
@@ -4215,76 +4215,20 @@ type T = A1 & A2 | A3 & A4 | A5 & A6 | A7 & A8 | A9 & A10[T10] |
   A61 & A62 | A63 & A64 | A65 & A66 | A67 & A68 | A69 & A70[T70]
 >>>
 type T =
-  A1 &
-    A2 |
-    A3 &
-    A4 |
-    A5 &
-    A6 |
-    A7 &
-    A8 |
-    A9 &
-    A10[T10] |
-    A11 &
-    A12 |
-    A13 &
-    A14 |
-    A15 &
-    A16 |
-    A17 &
-    A18 |
-    A19 &
-    A20[T20] |
-    A21 &
-    A22 |
-    A23 &
-    A24 |
-    A25 &
-    A26 |
-    A27 &
-    A28 |
-    A29 &
-    A30[T30] |
-    A31 &
-    A32 |
-    A33 &
-    A34 |
-    A35 &
-    A36 |
-    A37 &
-    A38 |
-    A39 &
-    A40[T40] |
-    A41 &
-    A42 |
-    A43 &
-    A44 |
-    A45 &
-    A46 |
-    A47 &
-    A48 |
-    A49 &
-    A50[T50] |
-    A51 &
-    A52 |
-    A53 &
-    A54 |
-    A55 &
-    A56 |
-    A57 &
-    A58 |
-    A59 &
-    A60[T60] |
-    A61 &
-    A62 |
-    A63 &
-    A64 |
-    A65 &
-    A66 |
-    A67 &
-    A68 |
-    A69 &
-    A70[T70]
+  A1 & A2 | A3 & A4 | A5 & A6 |
```

Comment:
Shouldn't unfold try to break it to separate lines as previously? This looks ok, but I am not sure about the expected bahaviour in case of long infix types

Reply:
no, `unfold` uses `afterInfix=many` which breaks a little more often than `fold`s `afterInfix=some` but not on every infix.

perhaps it could be improved but that would introduce changes to existing code which used `{Term,Type}.ApplyInfix` in other situations, not just `Type.{And,Or}`...